app/_src/using-mesh/managing-ingress-traffic/delegated.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Delegated gateways
+---
+
+Delegated gateways allow you to integrate existing API gateway solutions into your mesh.
+
+In delegated gateway mode, {{ site.mesh_product_name}} configures an Envoy sidecar for your API gateway.
+Handling incoming traffic is left to the API gateway while Envoy and
+{{ site.mesh_product_name }} take care of traffic leaving the gateway for the mesh.
+The non-{{ site.mesh_product_name }} gateway is in charge of policy like security or timeouts
+when it comes to incoming traffic and {{ site.mesh_product_name }} takes over
+after traffic leaves the gateway destinated for the mesh.
+
+At a technical level the delegated gateway sidecar is similar to any other sidecar
+in the mesh except that incoming traffic bypasses the sidecar and directly reaches the
+gateway.
+
+<center>
+<img src="/assets/images/docs/diagram-delegated-gateway-detailed@3x.jpg" alt="Delegated gateway" style="width: 387px; height: 452px; padding-top: 20px; padding-bottom: 10px;"/>
```

Comment:
**Suggestion**: Let’s do this for now:

```suggestion
<img src="/assets/images/docs/diagram-delegated-gateway-detailed@3x.jpg" alt="Delegated gateway" style="width: 387px;"/>
```

Specifically, the `height` property, now that it’s spelled correctly, actually distorts the image.